server/src/middlewares/auth.js

```diff
@@ -0,0 +1,28 @@
+import express from 'express';
+import jwt from 'jsonwebtoken';
+import user from '../models';
+
+/**
+ * @param {import('express').Request} req
+ * @param {import('express').Response} res
+ */
+export function authMiddleware(req, res, next) {
+    try {
+        const accessToken = req.cookies['access-token']
+        console.log(accessToken);
```

Comment:
Please remove console.log, it is bad practice to leave debug info like this as it quickly becomes just a noise that is making debugging harder.